packages/@react-aria/visually-hidden/src/VisuallyHidden.tsx

```diff
@@ -57,9 +57,9 @@ export function useVisuallyHidden(props: VisuallyHiddenProps = {}): VisuallyHidd
   } = props;
 
   let [isFocused, setFocused] = useState(false);
-  let {focusProps} = useFocus({
+  let {focusWithinProps} = useFocusWithin({
```

Comment:
This makes a lot more sense now, forgot `useFocus` ignores focus events on children. The `onFocusChange` I was seeing was from `useFocusRing` derp

Reply:
which can also take a 'within' parameter